pkg/diskmaker/diskmaker.go

```diff
@@ -253,15 +253,25 @@ func (d *DiskMaker) symLinkDisks(diskConfig *DiskConfig) {
 		for _, deviceNameLocation := range deviceArray {
 
 			symLinkDirPath := path.Join(d.symlinkLocation, storageClass)
+			var symLinkPath string
+			if deviceNameLocation.diskID != "" {
+				symLinkPath = getSymlinkPath(deviceNameLocation.diskID, symLinkDirPath)
+			} else {
+				symLinkPath = getSymlinkPath(deviceNameLocation.diskNamePath, symLinkDirPath)
+			}
 			baseDeviceName := filepath.Base(deviceNameLocation.diskNamePath)
```

Comment:
And we need to event if we are going to use device name for symbolic link?

Reply:
Also since we know of issues around using device name even when device id is available, I am thinking we should warn users when they use device name in `LocalVolume` CR.

Reply:
> And we need to event if we are going to use device name for symbolic link?

Added warning event when symbolic link source is device name. 

> Also since we know of issues around using device name even when device id is available, I am thinking we should warn users when they use device name in `LocalVolume` CR.

should this be a warning in the docs?

Reply:
yeah we should catch this warning in a doc I think.